query-engine/driver-adapters/executor/src/bench.ts

```diff
@@ -0,0 +1,104 @@
+import * as qe from "./qe";
+
+// pg dependencies
+import pgDriver from "pg";
+import * as prismaPg from "@prisma/adapter-pg";
+
+import { DriverAdapter } from "@prisma/driver-adapter-utils";
+import { webcrypto } from "node:crypto";
+
+import { recordReplay } from "./recordReplay";
+import prismaQueries from "../bench/queries.json";
```

Comment:
Easy, the queries to be benchmarked exist in the queries.json file

Reply:
I could have also added the schema in there so we don't need to feed it in the script. but that way we lose syntax highlighting and syntactic validation of the prisma.schema file.